onnxruntime/core/providers/cuda/nn/conv.h

```diff
@@ -160,8 +160,8 @@ class Conv : public CudaKernel {
 
  private:
   ConvAttributes conv_attrs_;
-
   mutable CudnnConvState<cudnnConvolutionFwdAlgoPerf_t> s_;
+  constexpr static auto kDefaultConvAlgo = CUDNN_CONVOLUTION_FWD_ALGO_IMPLICIT_PRECOMP_GEMM;
```

Comment:
>kDefaultConvAlgo [](start = 24, length = 16)

This variable is used only once. Is there any reason to keep it here?

Reply:
i think it makes the program more readable. Plus, the default algo may change in future. This way is a little easier to maintain.